compiler/src/iree/compiler/Codegen/LLVMCPU/test/illegal_configuration.mlir

```diff
@@ -173,3 +173,63 @@ hal.executable private @matmul_tensors {
     }
   }
 }
+
+// -----
+
+#config = #iree_codegen.lowering_config<tile_sizes = [[0, 7, 7, 64, 0, 0, 0], [6, 1, 7, 32, 0, 0, 0], [0, 0, 0, 0, 3, 3, 4]]>
+#translation = #iree_codegen.translation_info<CPUConvTileAndDecomposeExpert>
+#executable_layout = #hal.executable.layout<push_constants = 0, sets = [
+  #hal.descriptor_set.layout<0, bindings = [
+    #hal.descriptor_set.binding<0, storage_buffer>,
+    #hal.descriptor_set.binding<1, storage_buffer>,
+    #hal.descriptor_set.binding<2, storage_buffer>
+  ]>
+]>
```

Comment:
this test is failing at tip of tree since https://github.com/iree-org/iree/pull/10158 was merged too - midair collision :)
```
within split at /home/runner/actions-runner/_work/iree/iree/compiler/src/iree/compiler/Codegen/LLVMCPU/test/illegal_configuration.mlir:177 offset :5:27: error: unexpected error: unknown HAL attribute: executable.layout
#executable_layout = #hal.executable.layout<push_constants = 0, sets = [
                          ^
```
https://github.com/iree-org/iree/runs/7961662085?check_suite_focus=true

Reply:
oops, I reviewed the code, but I forgot that I added a test in the PR. I'll send out a fix! :P